js/package.json

```diff
@@ -46,6 +46,7 @@
     "json-loader": "^0.5.4",
     "mousetrap": "^1.6.0",
     "proj4": "^2.3.15",
+    "split.js": "^1.2.0",
```

Comment:
I saw it after a quick search.  It's a small library that does exactly what we want and nothing more, unlike any of the more common UI libraries.